tests/nopytest_ingestIndexReferenceCatalog.py

```diff
@@ -68,6 +70,10 @@ def testIngestTwoFilesTwoCores(self):
         self.checkAllRowsInRefcat(loader, skyCatalog1, config)
         self.checkAllRowsInRefcat(loader, skyCatalog2, config)
 
+    def testIngestTwoFilesTwoCores(self):
+        for withRaDecErr in [True, False]:
+            self._run_testIngestTwoFilesTwoCores(withRaDecErr)
```

Comment:
Can you please make this two separate calls? That way, if one fails, it will be much more obvious which one.